sections/04_tables.md

```diff
@@ -0,0 +1,15 @@
+# Tables
+
+Numbered tables can be included by using Markdown syntax to create the
+table and then adding a tag after the caption like this `{#tbl:label}`. The
+tables can be referenced in the text by using `@tbl:label`.
+
+Table @tbl:interest shows the relative search interest of the terms "Sci-Hub"
+and "LibGen" the week of June 25, 2017.
+
+week       search_term interest
```

Comment:
Let's use a markdown formatted table as generated by http://www.tablesgenerator.com/markdown_tables. I think this is more standard and robust than the format here